logging/logger.go

```diff
@@ -16,6 +18,7 @@ type Logger interface {
 	Sublogger(subname string) Logger
 	AddAppender(appender Appender)
 	AsZap() *zap.SugaredLogger
+	Write(*LogEntry)
```

Comment:
No idea why the linter doesn't complain about any of these. Can we explicitly document this with:
```
Unconditionally logs a LogEntry object. Specifically any configured log level is ignored.
```

I might follow up with adding a `Log(*LogEntry)` that _conditionally_ logs. So we can replace this [switch statement](https://github.com/viamrobotics/rdk/blob/927408d147a103384fd339c37df7ff5d3fa8eea0/robot/server/server.go#L435-L448) usage.